src/macros/macromanager.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <QObject>
+
+#include "macrorecorder.h"
+
+class MacroManager : public QObject {
```

Comment:
Well, there will be other classes for Macro playback and database storage. Rather than initializing them all in MixxxMainWindow, I created this class to encapsulate them.